productize.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
+
+LABELS=$(cat <<-END
+LABEL \
+com.redhat.component="redhat-sso-7-sso74-operator-rhel8-container"  \
+description="Red Hat Single Sign-On 7.4 Operator on OpenJDK OpenShift container image, based on the Red Hat Universal Base Image 8 Minimal container image" \
```

Comment:
Please remove reference to JDK since this is not relevant.

Reply:
Fixed.